ctru-rs/src/lib.rs

```diff
@@ -38,11 +44,13 @@ pub fn init() {
         // Only for panics in the main thread
         if main_thread == thread::current().id() && console::Console::exists() {
             println!("\nPress SELECT to exit the software");
-            let hid = services::hid::Hid::init().unwrap();
+
+            let hid = Hid::init().unwrap();
```

Comment:
A little out of scope for this PR, but maybe we should not unwrap here? Instead if there's an error, just return?

I don't have much opinion either way, but it stood out to me.

Reply:
Yeah, you’re right. Maybe I could print an error if the user is catching `stdout`, but panicking is a bad idea.